webpack/assets/javascripts/react_app/components/Editor/components/EditorOptions.js

```diff
@@ -2,14 +2,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import {
-  Button,
-  Icon,
-  OverlayTrigger,
-  FormControl,
-  Tooltip,
-} from 'patternfly-react';
+import { Button, Icon, FormControl } from 'patternfly-react';
```

Comment:
can you transform `Button` and `Icon` also to use `'@patternfly/react-core'`?

Reply:
out of the scope of this pr which is removing the overlaytrigger